index.html

```diff
@@ -1,12 +1,33 @@
-<h1>Sukonkina Arina</h1>
-<h2>About</h2>
+<!DOCTYPE html>
+<html>
+    <head>
+        <title><b>My profile</b></title>
+        <meta name="author" content="Arina Sukonkina">
+        <meta http-equiv="refresh" content="60">
```

Comment:
I suppose you're using this tag to help you during development, so you don't need to refresh the page when you make changes to it manually. If that's the case, you could explore some options in your text editor to create a local server that automatically updates your page when there are changes to the code. One example of this is VSCode's extension [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer).

If that's not the case, can you explain the use of this tag for me?